models/org_team.go

```diff
@@ -402,7 +413,10 @@ func UpdateTeam(t *Team, authChanged bool) (err error) {
 		}
 
 		if _, err = sess.Insert(&t.Units); err != nil {
-			sess.Rollback()
+			err = sess.Rollback()
+			if err != nil {
```

Comment:
This if statement is not necessary because no matter the outcome err will be returned